sos/plugins/ovirt_hosted_engine.py

```diff
@@ -59,10 +59,6 @@ def setup(self):
         all_setup_logs = glob.glob(self.SETUP_LOG_GLOB)
         all_setup_logs.sort(reverse=True)
```

Comment:
What is the point of making this list, sorting it, and then ... doing nothing with it apart from testing it has non-zero length?

This just looks like left-over garbage.